.github/workflows/modelScheme.py

```diff
@@ -0,0 +1,539 @@
+# Auto generated from model_card-schema.yaml by pythongen.py version: 0.9.0
+# Generation date: 2023-07-26T14:06:54
+# Schema: personinfo
```

Comment:
Can we not remove these comments? Or can we rename the Schema name and the url?

Reply:
I am currently looking into that. Modifying the default schema metadata.